### PR TITLE
Reduce duplicate stacktrace logging (Adapter, Receiver, Receiver) #4340

### PR DESCRIPTION
**Pull request by jkosternl**
Removed stacktrace from some logging locations. Kept message for handy details.

### CODE REVIEW

**Comment by tnleeuw** [comment]
I'm wondering if these are the right places to not log the stack traces.

Are the exceptions rethrown and caught and logged later in the chain?

Or are they originally logged earlier in the chain, then rethrown and caught again in these 2 places and logged?

**Comment by jkosternl** [comment]
> I'm wondering if these are the right places to not log the stack traces.
> Are the exceptions rethrown and caught and logged later in the chain?
> Or are they originally logged earlier in the chain, then rethrown and caught again in these 2 places and logged?

@tnleeuw the reason for this issue is that a problem is logged 3 times with all stacktraces added 3 times. Niels and me figured out that we can remove some stacktrace logging at a few locations, while keeping the message ID and the original message. 
We kept the complete stacktrace logging in Receiver.java at line: 1270 and 407.

File: core/src/main/java/nl/nn/adapterframework/core/Adapter.java [hunk 1]
```diff
@@ -240,18 +240,18 @@ public boolean configurationSucceeded() {
...
 	protected void warn(String msg) {
 		log.warn("Adapter [{}] {}", name, msg);
 		getMessageKeeper().warn(msg);
 	}
 
 	/**
-	 * sends a warning to the log and to the messagekeeper of the adapter
+	 * send an error to the log and to the messagekeeper of the adapter
 	 */
 	protected void addErrorMessageToMessageKeeper(String msg, Throwable t) {
```
Comment (nielsm5) [inline core/src/main/java/nl/nn/adapterframework/core/Adapter.java:253]: Deze methode wordt op andere plekken ook aangeroepen, nu laat hij nooit meer de stacktrace zien. Is dat wel verstandig?

Comment (jkosternl) [reply core/src/main/java/nl/nn/adapterframework/core/Adapter.java:253]: Ik dacht dat je deze aanraadde om aan te passen, in je chat op Teams. Maar idd wordt deze vaker gebruikt en dan zonder dat anderen de stacktrace loggen. Ik zal deze terugdraaien.

Comment (nielsm5) [reply core/src/main/java/nl/nn/adapterframework/core/Adapter.java:253]: Ik stel voor
`addErrorMessageToMessageKeeper("error processing message with messageId [" + messageId+"]: ",e);`
te vervangen door
`warn("error processing message with messageId [" + messageId+"]: " + e.getMessage());`

File: core/src/main/java/nl/nn/adapterframework/receivers/Receiver.java [hunk 2]
```diff
@@ -1652,7 +1652,7 @@ public void increaseRetryIntervalAndWait(Throwable t, String description) {
 		if (currentInterval>1) {
 			error(description+", will continue retrieving messages in [" + currentInterval + "] seconds", t);
 		} else {
-			log.warn("{}, will continue retrieving messages in [{}] seconds", description, currentInterval, t);
+			log.warn("{}, will continue retrieving messages in [{}] seconds. Details: {}", description, currentInterval, t != null ? t.getMessage() : "NA");
```
Comment (nielsm5) [inline core/src/main/java/nl/nn/adapterframework/receivers/Receiver.java:1655]: Deze warn om te zetten naar info

File: core/src/main/java/nl/nn/adapterframework/core/Adapter.java [hunk 3]
```diff
@@ -671,7 +671,7 @@ public PipeLineResult processMessageWithExceptions(String messageId, Message mes
 			}
 			processingSuccess = false;
 			incNumOfMessagesInError();
-			addErrorMessageToMessageKeeper("error processing message with messageId [" + messageId+"]: ",e);
+			log.warn("error processing message with messageId [{}]: {}", messageId, e.getMessage());
```
Comment (nielsm5) [inline core/src/main/java/nl/nn/adapterframework/core/Adapter.java:674]: Neen! De warn methode van deze klasse

Comment (jkosternl) [reply core/src/main/java/nl/nn/adapterframework/core/Adapter.java:674]: Aber Naturlich ;-)

**Review by nielsm5 — APPROVED** [review]

**Review by tnleeuw — APPROVED** [review]

**Pull request merged**